src/js/test/utilities/Logging.test.ts

```diff
@@ -52,7 +52,7 @@ describe("Logging Wrap", () => {
           assert(data.toString().includes("[test2] info: test2.2\n"));
           done();
         });
-      }, 10);
+      }, 100);
```

Comment:
add extra time for validating flush method, we need timeout because the flush process  happen in another process and it isn't possible handle promise